### ISSUE TITLE
Should I stop eating nuts?

### ISSUE THREAD

**Issue by contolini** [khandelwal/AskShashank#42]
Hi @khandelwal,

California is currently experiencing [one of the most severe droughts on record](http://ca.gov/drought/). @wpears has told me that the vast majority of water consumption in California is agricultural, not residential. The thirstiest crops in California are tree nuts, notably almonds and pistachios. Apparently, it takes an entire gallon of water to grow one almond.

Almonds and pistachios are delicious. I eat a lot of them. This causes me to feel guilty. Am I destroying California? Am I hurting America? Should I stop eating nuts?

Truly forever yours,

@contolini

**Comment by wpears** [comment]
Because everyone loves guilt-inducing web apps:
http://graphics.latimes.com/food-water-footprint/

Almonds aren't on there but they come to about ~25 gallons per ounce.
Whether ounces are a reasonable metric or not is another question. You could divide them by caloric content to get water per calorie (w/ WolframAlpha etc), but nutrient density should probably factor in too.

tl;dr toss out red meat and legumes and you can happily stuff yourself with almonds and save America

**Comment by jameshupp** [comment]
Another thing worth noting here is that almonds are increasingly replacing olives in crop production largely because they can be harvested mechanically. This reduces the need for labor and, obviously, cost. But almonds use more water than olives, and (by definition) employ fewer people during a time when (what is typically called) low-skill labor is still struggling.

So a possible market solution to your problem is to eat more olives in lieu of almonds rather than just eating fewer almonds. This is undoubtedly better for your health than a true deprivation diet. And if you can spread the word on this and help build momentum, the price of olives will increase while the price of almonds will decrease, thus counteracting the savings and leading to more employment opportunities in the agricultural sector while reducing water use.

**Comment by khandelwal** [comment]
![fullsizerender 28](https://cloud.githubusercontent.com/assets/636752/8359931/71d9948c-1b37-11e5-99c7-05bc99e32001.jpg)

**Comment by contolini** [comment]
Thank you for the feedback, everyone.

@khandelwal, in your drawing you show almonds requesting that I eat more olives. But, per @jameshupp's logic, this would result in fewer almonds being grown, reducing their population. I would think that sentient almonds, as you have clearly depicted, would want farmers to grow more of their own species by keeping consumer demand in their favor. This would increase almond production, providing an influx of foot soldiers for the inevitable bloodless  almond coup. I emphasize "bloodless" because I believe almonds to be peaceful seeds, unlike their violent cousin, the pecan, which are nearly impossible to crack open and don't even taste very good. Their primary use is in pie-making, where they are soaked in sugar and butter. Then, and only then, do they taste good.

In conclusion: Pecans are dumb.
